### PR TITLE
setGridField: undo column name truncation

### PR DESCRIPTION
**Pull request by dorschw**
## Status
- [x] Ready

## Description
Soon after merging #14469, we noticed that column names are only truncated by the web client when shown to user, but not when sent to the server. To not break column names longer than 255 chars, we decided to revert this truncation. 

## Minimum version of Cortex XSOAR
- [x] 5.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/ReleaseNotes/1_4_24.md [hunk 1]
```diff
@@ -1,4 +0,0 @@
-#### Scripts
```
Comment (DeanArbel) [inline Packs/CommonScripts/ReleaseNotes/1_4_24.md:1]: Don't remove this, as it's still a valid release note.

File: Packs/CommonScripts/ReleaseNotes/1_4_25.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#### Scripts
+
+##### SetGridField
+- Column names are no longer truncated after 255 chars (added on very recent v1.4.24)
```
Comment (DeanArbel) [inline Packs/CommonScripts/ReleaseNotes/1_4_25.md:4]:
```suggestion
- Add back support for column names with more than 255 chars (support was removed in v1.4.24).
```

**Review by DeanArbel — APPROVED** [review]
Please undo the RN delete.

File: Packs/CommonScripts/ReleaseNotes/1_4_24.md (hunk 1 above)
Comment (dorschw) [reply Packs/CommonScripts/ReleaseNotes/1_4_24.md:1]: unintentional, restoring.

**Pull request merged**